docs/content/docs/tutorials/eventbridge-example.md

```diff
@@ -0,0 +1,308 @@
+AWS services, and third- party software to consumer applications across your organization. EventBridge provides a simple
+and consistent way to ingest, filter, transform, and deliver events so you can build new applications quickly.
+
+In this tutorial you will learn how to create and manage a custom EventBridge [event
+bus](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-bus.html) and
+[rule](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-rules.html) to filter and forward messages to an SQS
+[target](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-targets.html) from an Amazon Elastic Kubernetes
+(EKS) deployment.
+
+## Setup
+
+Although it is not necessary to use Amazon Elastic Kubernetes Service (Amazon EKS) with ACK, this guide assumes that you
+have access to an Amazon EKS cluster. If this is your first time creating an Amazon EKS cluster, see [Amazon EKS
+Setup](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-eks-setup.html).
+For automated cluster creation using `eksctl`, see [Getting started with Amazon EKS -
+`eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) and create your cluster with
+Amazon EC2 Linux managed nodes.
+
+### Prerequisites
+
+This guide assumes that you have:
+
+- Created an EKS cluster with Kubernetes version 1.24 or higher.
+- AWS IAM permissions to create roles and attach policies to roles.
+- AWS IAM permissions to manages queues and send messages to a queue.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting
+    with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working
+    with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS
+    clusters.
+  - [Helm 3.8+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+  - [jq](https://stedolan.github.io/jq/download/) to parse AWS CLI JSON output
+
+### Install the ACK service controller for EventBridge
+
+Log into the Helm registry that stores the ACK charts:
+```bash
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+```
+
+Deploy the ACK service controller for Amazon EventBridge using the [eventbridge-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/eventbridge-chart). Resources should be created in the `us-east-1` region:
+
+```bash
+helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/eventbridge-chart --version=v0.0.3 --generate-name --set=aws.region=us-east-1
```

Comment:
How about going for v1.0.0? or even better curl the latest release tag on Github?

Reply:
Check out https://github.com/aws-controllers-k8s/community/pull/1736/files

Reply:
```
It will install the latest stable version of that chart until you specify '--devel' flag to 
also include development version (alpha, beta, and release candidate releases), 
or supply a version number with the '--version' flag.
```

Reply:
Removed `--version` in `Pipes` and `EventBridge` tutorials